src/gluonnlp/data/transforms.py

```diff
@@ -693,3 +696,299 @@ def __call__(self, sample):
             Detokenized text
         """
         return self._processor.DecodePieces(sample)
+
+
+class BasicTokenizer():
+    r"""Runs basic tokenization (punctuation splitting, lower casing, etc.).
+
```

Comment:
Let's make it clear in the documentation what tokenization steps are taken, because "BasicTokenizer" by its name is a bit vague:
- convert strings to unicode
- remove invalid chars (e.g. control chars) from the string
- tokenize JCK chars
- ...